### PR TITLE
Add helper command to run CLI commands on containers

### LINKED ISSUE #28: Running WP CLI in VIP Go containers
Running WP CLI in VIP Go containers is currently a faff; either running it from outside the container or from inside in a somewhat limited environment.

We've discussed running WP CLI commands via VIP CLI, piping `stdin`/`stdout`/`stderr`. We will need to consider how files can be passed for commands which specify file paths in an option.

### PR DESCRIPTION
**Pull request by joshbetz**
Runs a given command on a local container for the site specified.
If a container doesn't exist, output an error.

TODO:
* [x] Option to silence confirmation when we redirect stdout
* [ ] Deal with filenames as parameters

Fixes #28

### CODE REVIEW

**Comment by joshbetz** [comment]
@emrikol @david-binda @ryanmarkel The last item here is to deal with filenames as arguments. For example:

```
wp import <file>
```

```
wp some-command --file=<file>
```

Since the command is actually running in a different namespace, that file doesn't really exist. I have some ideas to make that work, but I wonder if it's worth doing right now.

Also note that piping stdio *does* work. For example:

```
wp users list --format=csv > users.csv
```

```
wp some-command < some-file.jpg
```

Thoughts? Do you think it's OK to merge this as it is right now and deal with that edge case later or do we run enough commands that require filenames to be passed as parameters, that we need to do it now?

**Comment by david-binda** [comment]
> Do you think it's OK to merge this as it is right now and deal with that edge case later

I don't think it's an edge case. I've just recently run a command for importing URLs for the legacy redirect plugin and it's quite a common thing for new site launches.

But I also believe that it might be good enough for now, but we also should keep in mind that we might eventually allow clients to run the commands on their own. How would we be dealing with files then? Would we require them to commit the files to the repository and thus then load them from inside the container so it would just work?

**Comment by joshbetz** [comment]
> I don't think it's an edge case. I've just recently run a command for importing URLs for the legacy redirect plugin and it's quite a common thing for new site launches.

We actually talked about this specific case on our call this week and decided we could rewrite the legacy redirect command to read the file from stdin instead, which would get around this issue.

> How would we be dealing with files then? 

@emrikol suggested a wp-cli helper that shows how to read files from stdin. As long as you only need to read read one file, this works fine.

File: src/bin/vip.js [hunk 1]
```diff
@@ -51,6 +52,38 @@ utils.getCredentials( ( err, user ) => {
...
+					];
+
+					if ( command.length < 1 ) {
+						run.push( 'bash' );
+					} else {
+						run = run.concat( command );
+
+						// TODO: Define this in wp-cli.yml
+						if ( "wp" == command[0] ) {
+							run.push( '--path=/var/www' );
```
Comment (nickdaugherty) [inline src/bin/vip.js]: Do we also need to add `--allow-root`?

**Comment by nickdaugherty** [comment]
For file paths, perhaps sandboxes should have a special mapped volume that points to a corresponding volume on the host - so that the directory structure matches. 

I'm thinking you put the file you want to import into `/home/cli-data/`, then when we invoke WP CLI, we can set the cwd to the mounted volume for that same directory.

File: src/bin/vip.js (hunk 1 above)
Comment (joshbetz) [inline src/bin/vip.js]: `-u 1001` sets the uid to 1001, which is the user that owns all of `/var/www` and, importantly, is not root. This seems mildly better than blindly passing `--allow-root` to all commands.

Comment (nickdaugherty) [inline src/bin/vip.js]: Ah, cool. Thanks for the clarification. That's certainly better.

**Comment by joshbetz** [comment]
> For file paths, perhaps sandboxes should have a special mapped volume that points to a corresponding volume on the host - so that the directory structure matches.

That's definitely another option, though it seems somewhat convoluted I'd say we should see how far we can get with reading files via `stdin` and deal with the issue of needing to reference files by name if/when it comes up.

**Comment by nickdaugherty** [comment]
Alright, sounds good to me.

**Pull request merged**